vizro-core/examples/visual-vocabulary/pages/_factories.py

```diff
@@ -110,3 +110,40 @@ def column_and_line_factory(group: str):
             make_code_clipboard_from_py_file("column_and_line.py"),
         ],
     )
+
+
+def waterfall_factory(group: str):
+    """Reusable function to create the page content for the column chart with a unique ID."""
+    return vm.Page(
+        id=f"{group}-waterfall",
+        path=f"{group}/waterfall",
+        title="Waterfall",
+        layout=vm.Layout(grid=PAGE_GRID),
+        components=[
+            vm.Card(
```

Comment:
This description feels a bit brief compared to other pages, but also I don't have anything to add to it really... Do you @huong-li-nguyen?

Reply:
I've suggested something :)